### ISSUE TITLE
Warn users about losing data from deletion of unpinned workspaces

### ISSUE THREAD

**Issue by jldec** [gitpod-io/gitpod#10486]
Unopened workspaces are automatically deleted afer a couple of weeks unless they are pinned.
It would be helpful to warn users before this happens, either in the UI, or via notification emails.

(Suggestion from support user [internal](https://app.frontapp.com/open/cnv_jdm0bwq).)

**Comment by svenefftinge** [comment]
Maybe easy to overlook, but it is there.
<img width="1354" alt="Screenshot 2022-06-06 at 21 08 22" src="https://user-images.githubusercontent.com/372735/172230120-52837632-9279-4316-8f76-67e7fe0a92f5.png">

**Comment by gtsiolis** [comment]
This message can become visually better and thus more noticeable, re-posting screenshot below from https://github.com/gitpod-io/gitpod/pull/10450#discussion_r890200627.

Alternatively, or additionally, we could improve the workspaces empty state or introduce a welcome modal to help educate users and make them more aware about the garbage collection[[1](https://www.gitpod.io/docs/life-of-workspace#garbage-collection)] and the task-based approach[[2](https://www.gitpod.io/blog/cloud-based-development-for-everyone)] of ephemeral workspaces.

| Improved garbage collection information | Improved workspaces empty state | Welcome modal |
|-|-|-|
| <img width="1440" alt="172178944-9910417a-9b90-4c40-99b1-3cd982c4ebf3" src="https://user-images.githubusercontent.com/120486/172239264-1bf6eb33-5f21-4fd8-afdc-27ae6655afc8.png"> | <img width="1440" alt="Welcome" src="https://user-images.githubusercontent.com/120486/172239341-ac548b92-95f4-4221-9f43-44ec47ae0900.png"> | <img width="1440" alt="Welcome-1" src="https://user-images.githubusercontent.com/120486/172239377-a4ac2dbd-df30-46b0-992e-78263a89cb56.png"> |

**Comment by axonasif** [comment]
What if we put it on the top of the list in a rather big noticeable info box?
Right now it might not be very much visible.
![Screenshot 2022-06-07 at 12-19-15 Workspaces — Gitpod](https://user-images.githubusercontent.com/39482679/172309999-fc018de2-c648-4cf4-91f5-a086834a6d97.png)

**Comment by YoungElPaso** [comment]
An email notification would be great as well!

**Comment by shivalikasingh95** [comment]
Yes please, an email notification would be really helpful before automatically deleting a workspace.
I've lost work a bunch of times because of this while using gitpod.
Github sends it for inactive code spaces and that's a really helpful reminder.
